CommandLine/emake/Server.cpp

```diff
@@ -127,6 +127,28 @@ class CompilerServiceImpl final : public Compiler::Service {
         if (author.empty())
           author = about.get("maintainer");
 
+        eyit represents = about.values.find("represents");
+        if (represents != about.values.end()) {
+          std::string repsStr = (represents->second)->data().get("build-platforms");

```

Comment:
I don't remember if `data()` returns a sentinel. Might be worth checking that it does.